templates/index.html

```diff
@@ -9,6 +9,12 @@
       <!-- Enable responsiveness on mobile devices-->
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    {% if page.description %}
```

Comment:
Wouldn't that fail for sections?

Reply:
I thought the index page would be considered a section? No need to add more

Reply:
I mean you probably understand it more than I do. All I can say is that it currently zola build and zola serve without issue with these changes.

Reply:
I'll have to doublecheck, it's odd

Reply:
I think its because this theme uses index directly as a template, where most themes currently have a base.html template, and the index template extends base.html...  Let me know what you find after you look it over.

Reply:
Did you ever figure out why the section is not an issue with this change?

Reply:
I just pushed another commit that should make it safe regardless of sections I believe.